<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="background">
      <property name="styleSheet">
       <string notr="true">background-color: #242424;

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="logs_label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>515</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Forte</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ffffff</string>
       </property>
       <property name="text">
        <string>Attendance Logs</string>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>716</x>
         <y>113</y>
         <width>3</width>
         <height>670</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #858585</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="liveVideoLabel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>169</y>
         <width>566</width>
         <height>326</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #D9D9D9;
border: 7px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>448</x>
         <y>32</y>
         <width>476</width>
         <height>69</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Forte</family>
         <pointsize>30</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ffffff</string>
       </property>
       <property name="text">
        <string>Attendance tracking System</string>
       </property>
      </widget>
      <widget class="QLabel" name="image">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>169</y>
         <width>246</width>
         <height>326</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #D9D9D9;
border: 7px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="login_btn">
       <property name="geometry">
        <rect>
         <x>974</x>
         <y>620</y>
         <width>158</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Forte</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #D9D9D9;
border-radius: 30px;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>113</y>
         <width>1370</width>
         <height>3</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #858585</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>74</x>
         <y>138</y>
         <width>94</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Forte</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ffffff</string>
       </property>
       <property name="text">
        <string>Live Video</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exit_btn">
       <property name="geometry">
        <rect>
         <x>1330</x>
         <y>10</y>
         <width>28</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../Downloads/cancel.png</normaloff>../../../../Downloads/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="minimize_btn">
       <property name="geometry">
        <rect>
         <x>1300</x>
         <y>10</y>
         <width>28</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../Downloads/minus (2).png</normaloff>../../../../Downloads/minus (2).png</iconset>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>1060</x>
         <y>240</y>
         <width>92</width>
         <height>169</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="imgName_label">
          <property name="font">
           <font>
            <family>Forte</family>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #ffffff</string>
          </property>
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="imgDate_label">
          <property name="font">
           <font>
            <family>Forte</family>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #ffffff</string>
          </property>
          <property name="text">
           <string>Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="imgStrand_label">
          <property name="font">
           <font>
            <family>Forte</family>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #ffffff</string>
          </property>
          <property name="text">
           <string>Strand:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="imgTime_label">
          <property name="font">
           <font>
            <family>Forte</family>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #ffffff</string>
          </property>
          <property name="text">
           <string>Time:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="imgGrade_label">
          <property name="font">
           <font>
            <family>Forte</family>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #ffffff</string>
          </property>
          <property name="text">
           <string>Grade:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="name_label">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>550</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>NAME</string>
       </property>
      </widget>
      <widget class="QLabel" name="grade_label">
       <property name="geometry">
        <rect>
         <x>265</x>
         <y>550</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>GRADE</string>
       </property>
      </widget>
      <widget class="QLabel" name="strand_label">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>550</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>STRAND</string>
       </property>
      </widget>
      <widget class="QLabel" name="time_label">
       <property name="geometry">
        <rect>
         <x>625</x>
         <y>550</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>TIME</string>
       </property>
      </widget>
      <widget class="QLabel" name="date_label">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>550</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>DATE</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="attendanceTableWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>571</y>
         <width>641</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget {
    background-color: transparent;
    color: #FFFFFF;
    border: none;
    gridline-color: transparent;
    outline: none;
    padding: 5px;
    font-size: 13px;
}

QHeaderView::section {
    background-color: transparent;
    color: #FFFFFF;
    font-size: 13px;
    border: none;
    padding: 5px;
    text-align: left;  /* Align header titles consistently */
}

QTableWidget::item {
    padding: 3px;
    border: none;
    text-align: left;  /* Important for item alignment */
}

QTableWidget::item:selected {
    background-color: transparent;
    color: #FFFFFF;
}

QTableWidget::item {
    selection-background-color: transparent;
    selection-color: #FFFFFF;
}

QScrollBar:vertical {
    width: 8px;
    background: transparent;
}

QScrollBar::handle:vertical {
    background-color: rgba(255, 255, 255, 0.4);
    border-radius: 4px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
}
</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Grade</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Strand</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Time</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
